extensions/yarn/PhylumExt.toml

```diff
@@ -3,6 +3,6 @@ description = "yarn package manager hooks"
 entry_point = "main.ts"
 
 [permissions]
-write = ["./package.json", "yarn.lock"]
-read = ["./package.json", "yarn.lock"]
+write = ["~/.cache", "~/.yarn", "./"]
+read = ["~/.cache", "~/.yarn", "./"]
```

Comment:
I'd restrict this to `~/.cache/node`, no reason to allow it to write to my Firefox cache for example.